test/e2e/l7networkpolicy_test.go

```diff
@@ -429,17 +435,39 @@ type L7LogAlertEntry struct {
 }
 
 type L7LogEntry struct {
-	EventType   string           `json:"event_type"`
-	DestIP      string           `json:"dest_ip"`
-	DestPort    int32            `json:"dest_port"`
-	Protocol    string           `json:"proto"`
-	AppProtocol string           `json:"app_proto,omitempty"`
-	Http        *L7LogHttpEntry  `json:"http,omitempty"`
-	Alert       *L7LogAlertEntry `json:"alert,omitempty"`
+	EventType           string           `json:"event_type"`
+	DestIP              string           `json:"dest_ip"`
+	DestPort            int32            `json:"dest_port"`
+	Protocol            string           `json:"proto"`
+	AppProtocol         string           `json:"app_proto,omitempty"`
+	PacketBytes         []byte           `json:"packet,omitempty"`
+	Http                *L7LogHttpEntry  `json:"http,omitempty"`
+	Alert               *L7LogAlertEntry `json:"alert,omitempty"`
+	expectedPacketRegex *regexp.Regexp
 }
 
-func (e *L7LogEntry) Equal(x *L7LogEntry) bool {
-	return reflect.DeepEqual(e, x)
+// Compares if two L7LogEntry are equal. They must be exactly equal if Packet is
+// in the same base. Otherwise, if either of the Packet is in base64, they equal
+// if Packet in base64 contains essential message in other Packet.
```

Comment:
does the comment match the current function?
There is nothing related to base64 in the function implementation.